mangum/adapter.py

```diff
@@ -71,6 +74,6 @@ def __call__(self, event: LambdaEvent, context: LambdaContext) -> dict:
             http_cycle = HTTPCycle(handler.scope, handler.body)
             http_response = http_cycle(self.app)
 
-            return handler(http_response)
+            return handler(http_response, self.text_mime_types)
```

Comment:
Could you place this in the `self.config` argument to the constructor for the handler instead? This would be similar to how `api_gateway_base_path` is used.

Reply:
This is a good point, I didn't notice that originally. I also moved `api_gateway_base_path` to a property with getter and setter referencing `.config` dictionary, because the way it was defined before would make changing the attribute itself meaningless - strings are immutable and all other functions/methods fetch that value from `config`, not from the `Mangum` attribute.

Reply:
Can continue discussion about property stuff on other comment.